vendor/vendor.json

```diff
@@ -49,10 +49,10 @@
 			"revisionTime": "2015-02-12T09:37:50Z"
 		},
 		{
-			"checksumSHA1": "EoJqr2ZG7jODFsCnKwrn4JWRS+Y=",
+			"checksumSHA1": "qUzU7BwMvcmsdozh1XVd0SAarAY=",
 			"path": "github.com/mvo5/libseccomp-golang",
-			"revision": "84e1d1c75beaa58be6a76d2fc94d95eb8c1583b6",
-			"revisionTime": "2017-06-14T13:46:31Z"
+			"revision": "e0e036d8f7d25f0c63e96896b99547e9d5f71617",
```

Comment:
This needs an update to the latest https://github.com/mvo5/libseccomp-golang/commits/master (f4de83b52afb3c19190eb65cc92429feaaf0e8b6) ?